main.nf

```diff
@@ -3,50 +3,56 @@
 
 // check parameters
 if (!params.inputDir) {
-    exit 1, "inputDir parameter is missing."
-}
-if (!params.outputDir) {
-    exit 1, "outputDir parameter is missing."
+    exit 1, 'inputDir parameter is missing.'
 }
 if ( !new File(params.inputDir).exists()) {
     exit 1, "The input folder does not exists."+params.inputDir+"\n"
 }
-if (!params.outPrefix) {
-    exit 1, "Output prefix parameter is missing."
+if (!params.outputDir) {
+    exit 1, "outputDir parameter is missing."
 }
-if (!params.debug) {
-    exit 1, "Debug prefix parameter is missing."
+if (!params.starTmpDir) {
+    exit 1, "starTmpDir parameter is missing."
+}
+
+if ( !new File(params.starTmpDir).exists()) {
```

Comment:
I suggest to be consistent with the spaces before and after the parentheses across the entire pipeline. I prefer no spaces, but it's a matter of style. But stick to one :)